services/prompter.js

```diff
@@ -103,7 +103,12 @@ async function Prompter(program, requests) {
         name: 'dbSchema',
         message: 'What\'s the database schema? [optional]',
         description: 'Leave blank by default',
-        when: answers => answers.dbDialect !== 'sqlite' && answers.dbDialect !== 'mongodb' && envConfig.dbDialect !== 'mongodb',
+        when: (answers) => {
+          // NOTICE: MongoDB, MySQL and SQLite do not require a Schema.
+          const skipDatabases = ['sqlite', 'mongodb', 'mysql'];
+          return !(skipDatabases.includes(answers.dbDialect)
```

Comment:
Maybe use `answer.dbDialect || envConfig.dbDialect`. If someone has a lingering .bashrc or .env file setting a MongoDB / MySQL db but now tries to generate a PGSQL project, they wouldn't be prompted for the schema with the current code. If the answer is set at runtime, the env variable should be ignored.

Reply:
Good catch!